public/controllers/loginController.js

```diff
@@ -0,0 +1,64 @@
+app.controller('loginController',function($scope,$timeout,requestService,buttonService){
```

Comment:
something dont understand - where is login logic and button logic, almost each button = request with action on response, I planned remove all button logic as buttonclick= function () {service.method}